peerdroid/src/main/java/rdx/works/peerdroid/data/PeerdroidLink.kt

```diff
@@ -205,20 +212,13 @@ internal class PeerdroidLinkImpl(
                 }
             }
             .onFailure { throwable ->
-                Timber.e("🛠️️ failed to create answer: ${throwable.message}")
+                Timber.e("🗼️ failed to create answer: ${throwable.message}❗")
                 terminateWithError()
             }
     }
 
     private suspend fun setLocalDescription(localSessionDescription: SessionDescriptionWrapper): Boolean {
-        return webRtcManager.setLocalDescription(localSessionDescription)
-            .onSuccess {
-                Timber.d("🛠️️ local description is set")
-            }
-            .onFailure { throwable ->
-                Timber.e("🛠️️ failed to set local description:${throwable.message} ")
-            }
-            .isSuccess
+        return webRtcManager.setLocalDescription(localSessionDescription).isSuccess
```

Comment:
we don't need to log anymore if this is not a success?

Reply:
yeah that was the issue that I logged the same thing twice. 
Because I get this log from the `WebRtcManager` class, therefore no need.